src/App.tsx

```diff
@@ -17,18 +48,29 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setInputValue={setInputValue}
```

Comment:
It's better not to pass a state setter function as a prop, wrap it in another function or a callback (see checklist 'pass initialize your state with props')